### ISSUE TITLE
[FR] Add support for Supercharged Links

### ISSUE THREAD

**Issue by Trinsec** [ozntel/file-tree-alternative#68]
I'd love it if support for [Supercharged Links](https://github.com/mdelobelle/obsidian_supercharged_links) could be added, then that'd colorize/modify the links in the file explorer to one's taste. The built-in file explorer of Obsidian supports this, it'd be neat if File Tree Alternative can do this too.

**Comment by sarantsatsral-qy** [comment]
Yes, please, this would be a great addition to the functionality of File Tree. Thanks!
